.github/workflows/releases.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   goreleaser:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Would it be fine to leave this as `ubuntu-latest`?

Reply:
`ubuntu-latest` accomplishes an upgrade from `mono` 4.6 to 6.8.
It's needed here to enable use of SHA256 signing in `signcode` (`-a sha256`) (see https://github.com/mono/mono/issues/7517)

A couple of alternatives:
- Use a different signing algorithm (md5 or sha1)
- Upgrade `mono` manually (see https://www.mono-project.com/download/stable/ for 18.04 instructions)